src/spikeinterface/core/baserecording.py

```diff
@@ -605,7 +605,7 @@ def _extra_metadata_to_folder(self, folder):
             if time_vector is not None:
                 np.save(folder / f"times_cached_seg{segment_index}.npy", time_vector)
 
-    def select_channels(self, channel_ids: list | np.array | tuple) -> "BaseRecording":
+    def _select_channels(self, channel_ids: list | np.array | tuple) -> "BaseRecording":
```

Comment:
```suggestion
    def select_channels(self, channel_ids: list | np.array | tuple) -> "BaseRecording":
```

What do you think on leaving this like this to avoid the navigation problem of clicking an recording object and being sent to snippets?

Reply:
I think it's better to have a function and overwrite it using inheritance,
I just followed the examples from other functions because I know that @samuelgarcia doesn't like inheritance and abstract classes :)

Reply:
OK, out of the scope then. I agree with you. I always wondered why we don't use abstact classes instead of this template pattern with private functions. Let's discuss this as a separate issue.